packages/core/content-manager/server/src/controllers/single-types.ts

```diff
@@ -192,15 +192,15 @@ export default {
   async countDraftRelations(ctx: any) {
     const { userAbility } = ctx.state;
     const { model } = ctx.params;
-
+    const { query = {} } = ctx.request;
     const entityManager = getService('entity-manager');
     const permissionChecker = getService('permission-checker').create({ userAbility, model });
 
     if (permissionChecker.cannot.read()) {
       return ctx.forbidden();
     }
 
-    const entity = await findEntity({}, model);
+    const entity = await findEntity(query, model);
     if (!entity) {
```

Comment:
```suggestion
    const sanitizedQuery = await permissionChecker.sanitizedQuery.read(query);

    const entity = await findEntity(sanitizedQuery, model);
```

I think we should sanitize the query as we do in the find